lib/manager/pipenv/extract.js

```diff
@@ -43,11 +44,31 @@ function extractFromSection(pipfile, section, registryUrls) {
     return [];
   }
   const specifierRegex = new RegExp(`^${specifierPattern}$`);
-  const deps = Object.entries(pipfile[section])
+  const oneSection = pipfile[section];
+
+  if (section === 'packages') {
+    Object.keys(oneSection).forEach(key => {
+      if (!is.string(oneSection[key]))
```

Comment:
Isn't it safer to check for an object? Or are we sure it can only be a string or an object?

Reply:
Done.